examples/counter-metrics/counter_metrics.md

```diff
@@ -10,6 +10,11 @@ For more information about configs and observing applications, see [Observing Ba
 
 ::: code ./examples/counter-metrics/counter_metrics.bal :::
 
+// Invoke the service three times using the cURL command below.
```

Comment:
```suggestion
// Invoke the service three times using the cURL command below.
```
Why are we saying three times at the same time? It will be clearer to have the 3 times explained separately one after the other.